collect_app/src/main/java/org/odk/collect/android/activities/CrashHandlerActivity.kt

```diff
@@ -0,0 +1,28 @@
+package org.odk.collect.android.activities
+
+import android.os.Bundle
+import androidx.activity.OnBackPressedCallback
+import org.odk.collect.crash_handler.CrashHandler
+import org.odk.collect.strings.localization.LocalizedActivity
+
+class CrashHandlerActivity : LocalizedActivity() {
```

Comment:
It references `MainMenuActivity`. I wanted to create some isolated crash handling tools that we could easily move to a Fragment or Compose based UI later so ended up with the Activity living in `collect_app` and handling the navigation.

Reply:
So maybe we could start `MainMenuActivity` without referencing it directly? For example by adding a custom category/intent name? You know I'm talking about marking the `MainMenuActivity` as the main activity and then start without using its class name.

Reply:
Totally see what you're saying, and it does make sense to be able to launch the "launch"/"start"/"home" Activity without an explicit reference. I started playing around with that, and then changed my mind: I still think the `CrashHandlerActivity` should live outside the `crash-handler` module and be a client of it rather than part of it. The reasoning for me here comes back to my earlier comment: 

> I wanted to create some isolated crash handling tools that we could easily move to a Fragment or Compose based UI later

Basically, it feels to me that (although these kinds of things are always guesses) that we're probably likely to end up changing navigation to the crash UI separately to changing the design itself. With that assumption, it makes sense to have `crash-handler` deal with recording and the actual UI elements, but let the client (here `CrashHandlerActiivty`) dictate the navigation as part of an actual app module (`collect_app`). Does that make sense?

Reply:
I'm not entirely convinced but I also do not have strong feelings to implement it in the way I described so ok.